### ISSUE TITLE
Unable to AddPageRoute to the default page

### ISSUE THREAD

**Issue by hishamco** [aspnet/Mvc#6338]
Razor Page by default looking for `index.cshtml`, using `AddPageRoute` i'm not able to route to `home.cshtml` using:
```csharp
services.AddMvc()
     .AddRazorPagesOptions(options =>
     {
          options.RootDirectory = "/Pages";
          options.AddPageRoute("/Home.cshtml", "/");
     });
```
Perhaps someone may ask why we need to home pages?
1- Assume I don't have `index.cshtml` at all
2- Look at CH9 as example, whenever a live event streaming, a new home page shows up instead of the default one

@pranavkm  any idea why this not work as expected?!!

**Comment by mikebrind** [comment]
It has only just been implemented and will be available in Preview 2.

https://github.com/aspnet/Mvc/issues/6325

**Comment by hishamco** [comment]
@mikebrind I already saw that issue and blog about the Friendly URLs in Razor Pages 😄 , my issue with routing `/`

**Comment by hishamco** [comment]
@pranavkm yep it's my mistake, but even if I used the relative file path without the extension, it doesn't route to the `/`

**Comment by pranavkm** [comment]
@hishamco - this is fixed post 2.0.0-preview2. You'll have to use `options.AddPageRoute("/Home", string.Empty);` to route to the pages root.

**Comment by hishamco** [comment]
Thanks a lot @pranavkm
